apps/server/src/modules/tool/controller/dto/response/custom-parameter.response.ts

```diff
@@ -0,0 +1,34 @@
+import { IsString } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class CustomParameterResponse {
+	constructor(props: CustomParameterResponse) {
```

Comment:
be consistant with the location of the constructor. on the top or the bottom. look how other classes do it.